packages/Tabs/src/components/Tab/Tab.styles.js

```diff
@@ -17,11 +24,11 @@ export const tabStyles = css`
   background-color: ${tokens.color.white};
   border: 0;
   border-bottom: ${tokens.spaceSm} solid transparent;
-  border-radius: 0;
+  border-radius: ${tokens.border.radius};
```

Comment:
I think you could still use the `box-shadow` property (with the appropriate token) in the `&:focus` block to make the link focus ring styling consistent?

Reply:
It's using the rawbutton focus styling though, i extracted this to the &:focus anyways though